policyengine_us/variables/gov/aca/slspc/slspc_rating_area.py

```diff
@@ -0,0 +1,43 @@
+from policyengine_us.model_api import * 
+from policyengine_us.parameters.gov.hhs.medicaid.geography import (
+    second_lowest_silver_plan_cost,
+)
+from policyengine_us.parameters.gov.hhs.medicaid.geography import (
+    aca_rating_areas,
+)
+class slspc_rating_area(Variable):
+    value_type = int
+    entity = Person
+    label = "Second-lowest ACA silver-plan cost rating area"
+    definition_period = YEAR
+
+    def formula(Person, period, parameters):
```

Comment:
```suggestion
    def formula(household, period, parameters):
```

Reply:
change functions to household 

When calling a different class, need to match the entity function of the class 
e.g.
`county_str` is a household level variable 
So the code will be 
```
county = household("county_str", period)
```